### PR TITLE
generation: add it and refactor tests

### PR DESCRIPTION
**Pull request by fenekku**
Just to have another pair of eyes before it is moved to inveniosoftware and released.

@ppanero

### CODE REVIEW

File: base32_identifier/base32.py [hunk 1]
```diff
@@ -81,6 +80,31 @@ def encode(number, split_every=0, min_length=0, checksum=False):
     return encoded
 
 
+def generate(length=8, split_every=0, checksum=False):
+    """Generate random base32 identifier.
```
Comment (ppanero) [inline base32_identifier/base32.py]: string instead of identifier, since now we are renaming the repo?

Comment (ppanero) [inline base32_identifier/base32.py]: Minor: string instead of identifier, since now we are renaming the repo?

**Review by ppanero — APPROVED** [review]
LGTM! :rocket:

File: base32_identifier/base32.py (hunk 1 above)
Comment (fenekku) [reply base32_identifier/base32.py]: done

**Pull request merged**